pkg/skaffold/docker/containers.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"context"
+	"io"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/pkg/stdcopy"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+)
+
+// ContainerRun runs a list of container.
+func (l *localDaemon) ContainerRun(ctx context.Context, out io.Writer, runs ...ContainerRun) error {
+	type c struct {
+		container containertypes.ContainerCreateCreatedBody
+		err       error
+	}
+
+	// Make sure containers are created concurrently to boost things up a bit.
+	containers := make(chan c, 2)
```

Comment:
should this not be `containers := make(chan c, len(runs))`?

Reply:
Added a comment to explain why